source/specify-documents-to-return.txt

```diff
@@ -0,0 +1,179 @@
+- ``sort()``: Specifies the sort order for the returned documents.
+- ``skip()``: Specifies the number of documents to skip before returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation,
+chain the ``limit()`` method to a ``find()`` method and pass the number
+of documents you want to return.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``"Italian"``, and limits the results to 5 documents:
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-limit-method
+      :end-before: end-limit-method
+      :language: java
+
+   .. output::
+
+      Isle Of Capri Resturant Italian
+      Arturo'S Italian
+      Patsy'S Italian Restaurant Italian
+      Piccola Venezia Italian
+      Roadhouse Restaurant Italian
+
+.. tip::
+
+   The preceding example returns the first five documents returned by the query, in
+   no particular order. The following section describes how to return the documents
+   in a specified sort order.
+
+.. _java-rs-sort:
+
+Sort
+----
+
+To return documents in a specified order, chain the ``sort()`` method to a
+``find()`` method. The ``sort()`` method takes one parameter: a sort direction
+method. To specify the sort direction, pass either the ``ascending()`` or ``descending``
+method to the ``sort()`` method. Then, pass the field value you want to sort by.
+The ``ascending()`` method sorts values from lowest to highest, and
+the ``descending()`` method sorts them from highest to lowest. If you don't specify either
+direction, the method defaults to sorting in ascending order.
+
+The following example returns all documents with the ``cuisine`` value of
+``"Italian"``, sorted in ascending order:
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-sort-method
+      :end-before: end-sort-method
+      :language: java
+
+   .. output::
+
+      (Lewis Drug Store) Locanda Vini E Olii
+      101 Restaurant And Bar
+      44 Sw Ristorante & Bar
+      900 Park
+      A Voce
+      ...
+      Zucchero E Pomodori
+
+Skip
+----
+
+To skip a specified number of documents before returning your query results,
+chain the ``skip()`` method to a ``find()`` method and pass the number of documents to skip. The
+``skip()`` method ignores the specified number of documents in your query
+results and returns the rest.
+
+The following example returns all documents that have a ``borough`` field value
+of ``"Manhattan"``, and skips the first 10 documents:
```

Comment:
If you decide to use monospace for `5` like in the limit example, can also do so here: 
```suggestion
of ``"Manhattan"``, and skips the first ``10`` documents:
```